index.html

```diff
@@ -0,0 +1,380 @@
+              </li>
+              <li>
+                <a href="https://twitter.com/M_ayienga"
+                  ><i class="fab fa-github"></i>
+                </a>
+              </li>
+              <li>
+                <a href="marlymayienga@gmail.com"
+                  ><i class="fab fa-angellist"></i
+                ></a>
+              </li>
+            </ul>
+            <img class="img_three" src="./assets/img_three.png" alt="" />
+          </div>
+        </div>
+        <img class="img_four" src="./assets/img_four.png" alt="" />
+      </section>
+    </main>
+    <section class="works">
+      <div class="work-item">
+        <div>
+          <img
+            class="images"
+            src="./assets/project_img1.png"
+            alt="project_img1"
+          />
+        </div>
+        <div class="work-item-detail">
+          <h5 class="project-headline">Tonic</h5>
+          <div class="project-tags">
+            <div>
+              <ul>
+                <li>Canopy</li>
+                <li>
+                  <span><i class="fas fa-circle"></i></span> BackEnd Dev
+                  <span><i class="fas fa-circle"></i></span>
+                </li>
+                <li>2015</li>
+              </ul>
+            </div>
+          </div>
+          <p class="project-description">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="container-skill">
+            <ul>
+              <li><button class="skill-button">html</button></li>
+              <li><button class="skill-button">css</button></li>
+              <li><button class="skill-button">Javascript</button></li>
+            </ul>
+          </div>
+          <button class="see-project">See Project</button>
+        </div>
+      </div>
+      <div class="work-item">
+        <div class="work-item-detail">
+          <h5 class="project-headline">Multi-Post Stories</h5>
+          <div class="project-tags">
+            <ul>
+              <li>FACEBOOK</li>
+              <li>
+                <span><i class="fas fa-circle"></i></span> FullStack Dev
+                <span><i class="fas fa-circle"></i></span>
+              </li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p class="project-description">
+            Experimental content creation feature that allows users to add to an
+            existing story over the course of a day without spamming their
+            friends.
+          </p>
+          <div class="container-skill">
+            <ul>
+              <li><button class="skill-button">html</button></li>
+              <li><button class="skill-button ruby">Ruby on Rails</button></li>
+              <li><button class="skill-button">css</button></li>
+              <li><button class="skill-button">Javascript</button></li>
+            </ul>
+          </div>
+          <button class="see-project">See Project</button>
```

Comment:
- [ ] Please add button interaction styling for this button. It should change its colors when: Hover, Active and Pressed. Don't forget to change its cursor to pointer on hover.